src/providers/onelogin.js

```diff
@@ -0,0 +1,19 @@
+export default function OneLogin(options) {
+  return {
+    id: "onelogin",
+    name: "OneLogin",
+    type: "oauth",
+    version: "2.0",
+    scope: "openid profile name email",
+    params: { grant_type: "authorization_code" },
+    // These will be different depending on the Org.
+    accessTokenUrl: `https://${options.domain}/oidc/2/token`,
+    requestTokenUrl: `https://${options.domain}/oidc/2/auth`,
+    authorizationUrl: `https://${options.domain}/oidc/2/auth?response_type=code`,
```

Comment:
Hey 👋 no worries! Unfortunately no 😞, in OneLogin every company has their own instance, and an instance is tied to a subdomain, so it would be <subdomain>.onelogin.com and OneLogin doesn't provide a default instance.

Reply:
Okey, thanks for the clarification 😊